### PR TITLE
New tutorial for cleaning SARS-CoV-2 sequencing data by removing human reads

### PR DESCRIPTION
**Pull request by wm75**

### CODE REVIEW

**Review by bebatut — COMMENTED** [review]
I did not test the tutorial itself, I just put some suggestions

File: topics/sequence-analysis/tutorials/human-reads-removal/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+# Read trimming and mapping
+
+In order to learn, which of the reads of each of the input samples are of likely human origin we need to map them to the human reference genome. To improve the quality of this step we will also trim low-quality bases from the ends of all reads before passing them to the read mapping software.
+
+> ### {% icon hands_on %} Hands-on: Trim reads and map them to the human reference genome
+>
+> 1. {% tool [Trimmomatic](toolshed.g2.bx.psu.edu/repos/pjbriggs/trimmomatic/trimmomatic/0.38.0) %} with default settings and:
+>    - *"Single-end or paired-end reads?"*: `Paired-end (as collection)`
+>      - *"Select FASTQ dataset collection with R1/R2 pair"*: the collection of sequenced reads input datasets
```
[the remark this answers is not in the document]
Comment (wm75) [reply topics/sequence-analysis/tutorials/human-reads-removal/tutorial.md:123]: This is not a regular data param with this icon - selecting a collection is the only option if you select *Paired-end (as collection)* above.
So I don't think the icon should be added here.

File: topics/sequence-analysis/tutorials/human-reads-removal/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+> 2. {% tool [Map with BWA-MEM](toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa_mem/0.7.17.2) %} with the following parameters:
+>    - *"Will you select a reference genome from your history or use a built-in index?"*: `Use a built-in genome index`
+>      - *"Using reference genome"*: `Human Dec. 2013 (GRCh38/hg38)(hg38)`
+>
+>        The exact name of this version of the human reference genome may vary between Galaxy servers. Start typing `hg38` in the select box to reveal available choices on your server.
+>    - *"Single or Paired-end reads"*: `Paired Collection`
+>      - *"Select a paired collection"*: the `paired` output of **Trimmomatic**
```
[the remark this answers is not in the document]
Comment (wm75) [reply topics/sequence-analysis/tutorials/human-reads-removal/tutorial.md:130]: Same as in the Trimmomatic case.

File: topics/sequence-analysis/tutorials/human-reads-removal/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Get data
+
+As always, it is best to give each analysis you are performing with Galaxy its own history, so let's start with preparing this:
+
+> ### {% icon hands_on %} Hands-on: Data upload to Galaxy from NCBI SRA
```
[the remark this answers is not in the document]
Comment (wm75) [reply topics/sequence-analysis/tutorials/human-reads-removal/tutorial.md:69]: Uha, thanks for catching :)

**Review by bebatut — APPROVED** [review]
Let's merge this 1st version and see if we need to fix it later. Thanks @wm75!

**Pull request merged**